code/game/machinery/cryopod.dm

```diff
@@ -192,12 +192,20 @@
 		/obj/item/device/paicard,
 		/obj/item/weapon/gun,
 		/obj/item/weapon/pinpointer,
-		/obj/item/clothing/suit,
 		/obj/item/clothing/shoes/magboots,
 		/obj/item/areaeditor/blueprints,
 		/obj/item/clothing/head/helmet/space,
+		/obj/item/clothing/suit/space,
+		/obj/item/clothing/suit/armor/laserproof,
+		/obj/item/clothing/suit/armor/reactive,
+		/obj/item/weapon/tank,
+		/obj/item/weapon/defibrillator/compact,
+		/obj/item/weapon/reagent_containers/hypospray,
```

Comment:
this is going to save every single hypospray known to man; guess what medipens are.